keyboards/kprepublic/bm40hsrgb/keymaps/skewwhiffy/readme.md

```diff
@@ -0,0 +1,25 @@
+# Skewwhiffy keymap
+
+## Overview
+
+A small-form ergonomic Colemak-based multi-layered layout, optimized for software engineering. Programmed to emulate a standard UK ISO keyboard.
+
+Keyboard layout can be viewed [here](http://www.keyboard-layout-editor.com/#/gists/470f01e934c27d41491dea9ef5c6fde6)
+
+![Keyboard layout](./keyboard-layout.png)
```

Comment:
Above comment wrt. to layout image.

Reply:
I've also squashed my PR so that the image is no longer in git at all.